awswrangler/s3/_write.py

```diff
@@ -55,13 +55,14 @@ def _validate_args(
     description: Optional[str],
     parameters: Optional[Dict[str, str]],
     columns_comments: Optional[Dict[str, str]],
+    distributed: Optional[bool] = False,
```

Comment:
Very minor nitpick, but I'm curious why you're passing this parameter rather than just using `config.distributed` in the condition below.

Reply:
It was causing a circular dependency. But I'll have a look for other ways to resolve it